guillotina/content.py

```diff
@@ -700,6 +700,7 @@ def get_all_behavior_interfaces(content) -> list:
         id=new_id,
         creators=context.creators,
         contributors=context.contributors,
+        check_security=check_permission,
```

Comment:
Do you guys think that's ok? IMHO yes, but I'd appreciate your confirmation

Reply:
Yes, this is good.